src/Ch9/Ch9.Shared/App.xaml.cs

```diff
@@ -25,223 +26,273 @@
-		}
+            resources.Add("StatusBarDouble", (double)statusBarHeight);
+            resources.Add("StatusBarThickness", new Thickness(0, statusBarHeight, 0, 0));
+            resources.Add("StatusBarGridLength", new GridLength(statusBarHeight, GridUnitType.Pixel));
+        }
 
-		private void ConfigureOrientation()
-		{
-			if (DeviceInfo.Idiom == DeviceIdiom.Phone)
-			{
-				DisplayInformation.AutoRotationPreferences = DisplayOrientations.Portrait;
-			}
-		}
+        private void ConfigureOrientation()
+        {
+            if (DeviceInfo.Idiom == DeviceIdiom.Phone)
+            {
+                DisplayInformation.AutoRotationPreferences = DisplayOrientations.Portrait;
+            }
 
-		/// <summary>
-		/// Sets the visibility of the system UI's back button based on the navigation service.
-		/// </summary>
-		private void ConfigureSystemBackVisibility()
-		{
-			var navigationService = ServiceProvider.GetInstance<IStackNavigationService>();
-
-			void OnNavigated(IStackNavigationService sender, OnNavigatedEventArgs args)
-			{
-				SystemNavigationManager.GetForCurrentView().AppViewBackButtonVisibility = sender.CanGoBack
-					? AppViewBackButtonVisibility.Visible
-					: AppViewBackButtonVisibility.Collapsed;
-			}
+            var simpleOrientationSensor = SimpleOrientationSensor.GetDefault();
 
-			navigationService.OnNavigated += OnNavigated;
-		}
+            if (simpleOrientationSensor != null)
+            {
+                simpleOrientationSensor.OrientationChanged += OrientationChanged;
+            }
 
-		/// <summary>
-		/// Hooks the system back button with the navigation service.
-		/// </summary>
-		private void ConfigureBackRequests()
-		{
-			void OnBackRequested(object sender, BackRequestedEventArgs e)
-			{
+            void OrientationChanged(SimpleOrientationSensor sender, SimpleOrientationSensorOrientationChangedEventArgs args)
+            {
+                try
+                {
+                    var isLandscape = args.Orientation.IsOneOf(
+                        SimpleOrientation.Rotated270DegreesCounterclockwise,
+                        SimpleOrientation.Rotated90DegreesCounterclockwise
+                    );
+
+                    if ((_rootFrame.Content as FrameworkElement)?.DataContext is ShowPageViewModel showPage &&
+                        showPage.Show.SelectedEpisode != null)
+                    {
+                        ToVideoFullWindow(showPage.Show, isLandscape);
+                    }
+                    else if ((_rootFrame.Content as FrameworkElement)?.DataContext is MainPageViewModel mainPage &&
+                             mainPage.Show.SelectedEpisode != null)
+                    {
+                        ToVideoFullWindow(mainPage.Show, isLandscape);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    this.Log().ErrorIfEnabled(() => $"Error in OrientationChanged subscription: {ex.ToString()}");
```

Comment:
You probably don't need the `.ToString()` because you're in a string interpolation.